src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -5763,6 +5798,93 @@ class BodyVisitor : public CommonVisitor<BodyVisitor> {
         return res;
     }
 
+    void string_compile_time_evaluation(std::string &s_var, ASR::expr_t* arg_seperator, 
```

Comment:
```suggestion
    void eval_partition(std::string &s_var, ASR::expr_t* arg_seperator, 
```
or something similar